src/features/User/Ui/CreateBooking/CreateBooking.tsx

```diff
@@ -0,0 +1,240 @@
+})
+      .then((response) => {
+        // console.log(response);
+        // console.log(response?.data?.data?.booking?._id);
+        // setBookingId(response.data.data.booking._id)
+const bookingId = response.data.data.booking._id
+        console.log(bookingId );
+
+        navigate(`/user/home/booking-details/${bookingId}`);
+      })
+      .catch((error) => {
+
+      });
+    //   try {
+    //     const response = await axios.post(createBookingUrl, requestBody, {
+    //       headers: requestHeaders,
+    //     });
+    //     // console.log(response);
+    //     // console.log(response.data.data.booking);
+     
+    //     if (response?.data?.data?.booking) {
+    //       setBookingDetails(response.data.data.booking);
+    //       console.log(response.data.data.booking);
+    //     } else {
+    //       console.log("La réponse ne contient pas de données de réservation.");
+    //     }
+    //     if (response?.data?.data?.booking?._id) {
+    //       setBookingId(response.data.data.booking._id);
+    //       console.log(response.data.data.booking._id);
+    //       console.log(bookingId);
+    //     } else {
+    //       console.log("La réponse ne contient pas de données de réservation.");
+    //     }
+    //     navigate(`/user/home/booking-details/${bookingId}`);
+    //   } catch (error) {
+    //     console.error(error);
+    //   }
+     }
+  };
+  // useEffect(() => {
+  //   // console.log(bookingDetails);
+  //   // console.log(bookingDetails._id);
+  //   console.log(bookingId);
+  // }, []);
+
+  const formatDate = (date) => {
+    if (!(date instanceof Date)) {
+      date = new Date(date); // Crée une nouvelle instance de Date si ce n'est pas déjà le cas
+    }
+
+    const year = date.getFullYear();
+    const month = String(date.getMonth() + 1).padStart(2, "0");
+    const day = String(date.getDate()).padStart(2, "0");
+    return `${year}-${month}-${day}`;
+  };
+
+  return (
+    <Box
+      component="form"
+      noValidate
+      onSubmit={handleSubmit(onSubmit)}
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+      }}
+    >
+      <Typography component="h6" variant="h5">
+        create a booking
+      </Typography>
+      <Typography sx={{ ml: 1 }} component="h6" variant="h6">
+        Pick a Date
+      </Typography>
+      <Box
+        sx={{
+          display: "flex",
+          alignItems: "center",
+        }}
+      >
+        <Box display="flex" alignItems="center"></Box>
+
+        <MyDateRangePicker
+          onDateChange={handleDateChange}
+          ref={dateRangePickerRef}
+          // {...register("startDate", { required: 'Start date is required' })}
+          // {...register("endDate", { required: 'End date is required' })}
+        />
+      </Box>
+
+      <Typography
+        className={styles.headParag}
+        sx={{ ml: 1 }}
+        component="h6"
+        variant="h6"
+      >
+        You will pay <strong>$00USD </strong>
```

Comment:
This value need to be dynamic based on the room price and number of people and date range.